packages/shared/src/features/onboarding/shared/FunnelStepper.tsx

```diff
@@ -135,27 +136,44 @@ export const FunnelStepper = ({
     }
   };
 
+  const layout = useMemo(() => {
+    const hasBanner = !!step?.parameters?.banner?.content;
+    const hasHeader = !hasBanner && stepsWithHeader.includes(step.type);
```

Comment:
I think we should always show it regardless of the current step having a header too. Wdyt? Maybe we need to check this with product team.

Reply:
In design we don't have header when there is a banner and those element are not good together from UI perspective. However, I'll ask to design team ✔️

Reply:
I agree with that. Just that if I remember correctly, their goal of having this banner was to push everything down to make things closer to the thumb position. Let's see what have to say 👀